opencti-platform/opencti-front/playwright.config.ts

```diff
@@ -51,17 +51,30 @@ export default defineConfig({
   timeout: 200000,
   /* Configure projects for major browsers */
   projects: [
-    { name: 'setup', testMatch: /.*\.setup\.ts/ },
+    {
+      name: 'setup',
+      testMatch: /.*\.setup\.ts/
+    },
+    {
+      name: 'init data',
+      testMatch: "init.data.ts",
+      use: {
+        ...devices['Desktop Chrome'],
+        storageState: 'tests_e2e/.setup/.auth/user.json',
+      },
+      dependencies: ['setup'],
+    },
```

Comment:
Add a new project to be able to run the test "Init data" after "Authenticate" but before all other tests